package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ably-common",
+  "version": "1.0.0",
```

Comment:
Perhaps start this off at `1.2.0` to represent protocol version as it stands right now?

Reply:
Good idea aee5762